pyvista/core/pointset.py

```diff
@@ -106,29 +106,84 @@ class PolyData(_vtk.vtkPolyData, PointSet, PolyDataFilters):
     - Using vertices and faces
     - From a file
 
+    Parameters
+    ----------
+    var_inp : vtk.PolyData, iterable, optional
+        Flexible input type.  Can be a ``vtk.PolyData``, in which case
```

Comment:
Either `vtk.vtkPolyData` or `PolyData` for our own.
```suggestion
    var_inp : vtk.vtkPolyData, iterable, optional
        Flexible input type.  Can be a ``vtk.vtkPolyData``, in which case
```